.eslintrc.js

```diff
@@ -19,7 +19,8 @@ module.exports = {
         '@typescript-eslint/no-useless-constructor': 'error',
         'no-empty-function': 'off',
         '@typescript-eslint/no-empty-function': 'error',
-        'import/prefer-default-export': 'off'
+        'import/prefer-default-export': 'off',
+        "import/no-extraneous-dependencies": ["error", {"devDependencies": ["**/*.test.ts"]}]
```

Comment:
May want to format this file you are using " while other lines are '

Also just to confirm what does this line do?

Reply:
Converting `"` to `'`. 

In test files we would get a lint error for `no-extraneous-dependencies` if the test file imports a library that is included in `devDependencies` instead of `dependencies`. This rule turns off that error for test files.